pkg/datarepair/checker/checker_test.go

```diff
@@ -151,12 +151,12 @@ func TestOfflineNodes(t *testing.T) {
 	overlayServer := mocks.NewOverlay(nodes)
 	limit := 0
 	interval := time.Second
-	irrdb, err := irreparabledb.New("sqlite3://file::memory:?mode=memory&cache=shared")
+	// creating in-memory db and opening connection
+	db, err := satellitedb.NewDB("sqlite3://file::memory:?mode=memory&cache=shared")
```

Comment:
You need to use close db (`db.Close()`) e.g. https://github.com/storj/storj/blob/master/pkg/accounting/tally/tally_test.go#L69